source/html/renderer/src/host-config.ts

```diff
@@ -69,62 +79,85 @@ function parseColorPaletteDefinition(obj: any): IContainerStyleDefinition {
     } : null;
 }
 
-export interface IAdaptiveCardConfig {
-    allowCustomStyle: boolean
+export class AdaptiveCardConfig {
+    allowCustomStyle: boolean;
 }
 
-function parseAdaptiveCardConfiguration(obj: any): IAdaptiveCardConfig {
+function parseAdaptiveCardConfiguration(obj: any): AdaptiveCardConfig {
     return obj ? {
         allowCustomStyle: obj["allowCustomStyle"]
     } : null;
 }
 
-export interface IImageConfig {
-    size: Enums.Size
+export class ImageConfig {
+    size: Enums.Size;
+
+    toJSON() {
+        return {
+            size: Enums.Size[this.size]
+        }
+    }
 }
 
-function parseImageConfiguration(obj: any): IImageConfig {
-    return obj ? {
+function parseImageConfiguration(obj: any): ImageConfig {
+    return obj ? Object.assign(new ImageConfig(), {
         size: obj["size"]
-    } : null;
+    }) : null;
 }
 
-export interface IImageSetConfig {
+export class ImageSetConfig {
     imageSize: Enums.Size
     maxImageHeight: number
+
+    toJSON() {
+        return {
+            imageSize: Enums.Size[this.imageSize],
+            maxImageHeight: this.maxImageHeight
+        }
+    }
 }
 
-function parseImageSetConfiguration(obj: any): IImageSetConfig {
-    return obj ? {
+function parseImageSetConfiguration(obj: any): ImageSetConfig {
```

Comment:
Since ImageSetConfig is now a class, this method should be moved to the class and should be renamed into simply "parse"